src/context/like-video-context.jsx

```diff
@@ -24,6 +26,12 @@ const LikedVideoProvider = ({ children }) => {
         );
         if (response.status === 201) {
           setLikedVideos(response.data.likes);
+          setToastStyles("alert alert-info")
+          setToastMsg("video is added to liked videos")
```

Comment:
Just a suggestion: Since it's a video library itself, you can write `Added to liked videos` instead of using video word twice. Also, Make the first `v` capital

Reply:
okay, i will the changes